Extensions/xcloud.js

```diff
@@ -12,6 +12,15 @@ XKit.extensions.xcloud = new Object({
 	username: "",
 	password: "",
 
+
+	get_xcloud_url: function() {
+		if (XKit.extensions.xcloud.useoldserver === true) {
```

Comment:
Should we use that convention?
I'm not sure if I want true-ish values to be true.